packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx

```diff
@@ -0,0 +1,31 @@
+import BasicFolderTree from './BasicFolderTree?dev-site-example';
+
+# Selection
+
+### Description
+
+The following example demonstrates row selection.
+
+### Properties required
+
+Selection state is controlled by the consuming component.
+The following FolderTreeItem props are required for row selection:
+* **isSelected**: Identifies whether or not the FolderTreeItem is currently selected. 
```

Comment:
Same as above, probably should be links to the props table.

Reply:
[4b462af](https://github.com/cerner/terra-framework/pull/1973/commits/4b462af48977e5fb036f4ed1a0a9666f2918c9de)